src/test/java/seedu/address/storage/JsonSerializableTaskBookTest.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.commons.util.JsonUtil;
+import seedu.address.model.TaskBook;
+import seedu.address.testutil.TypicalTasks;
+
+public class JsonSerializableTaskBookTest {
+
+    private static final Path TEST_DATA_FOLDER = Paths.get("src", "test", "data", "JsonSerializableTaskBookTest");
+    private static final Path TYPICAL_TASKS_FILE = TEST_DATA_FOLDER.resolve("typicalTasksTaskBook.json");
+    private static final Path INVALID_TASK_FILE = TEST_DATA_FOLDER.resolve("invalidTasksTaskBook.json");
+
```

Comment:
Is it better to test the case where there are duplicate tasks too?
e.g. `DUPLICATE_TASK_FILE`

Reply:
Thanks for the suggestion!! Added that test case!